### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by jmandel** [PaulWoitaschek/Voice#1617] [General]
Just wanted to start a thread saying thanks for creating such a focused, simple, and powerful app as free software! I've had a lovely initial experience with Voice.

**Comment by Robin-Velas** [comment]
I agree, it is a wonderful app! Thank you Paul ❤️

**Comment by Greenheart** [comment]
Yeah, a warm thank you to everyone involved in creating and continously improving this app! 💚

It's a marvellous example of why and how free (as in freedom) software creates so much more value compared to proprietary "alternatives".

Thanks for the inspiration to continue creating free software!

**Comment by ghost** [comment]
I'm new to audiobooks, and I was worried that I wouldn't be able to find a good, FOSS app to use. I'm so thankful that I came across this one. It's so simple and intuitive. Thank you so much for making it!

**Comment by azzamsa** [comment]
I think you need to enable Github Sponsor @PaulWoitaschek 🍨

**Comment by Pixelshiftarr** [comment]
Also wanna add my thanks! I love how beautiful and simple this app is. It does exactly what you would expect from such an app and isn't overloaded with tons of additional features. Love the simplicity! Lots of thanks!
